From 52082c2e99d0962ae7576c866f0b11de26c3526a Mon Sep 17 00:00:00 2001
From: Greg Hackmann <ghackmann@google.com>
Date: Mon, 10 Mar 2014 14:21:30 -0700
Subject: [PATCH 342/634] power: wakeup_reason: rename irq_count to irqcount

On x86, irq_count conflicts with a declaration in
arch/x86/include/asm/processor.h

Change-Id: I3e4fde0ff64ef59ff5ed2adc0ea3a644641ee0b7
Signed-off-by: Greg Hackmann <ghackmann@google.com>
---
 kernel/power/wakeup_reason.c | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/kernel/power/wakeup_reason.c b/kernel/power/wakeup_reason.c
index caf4421..188a6bf 100644
--- a/kernel/power/wakeup_reason.c
+++ b/kernel/power/wakeup_reason.c
@@ -30,7 +30,7 @@
 
 #define MAX_WAKEUP_REASON_IRQS 32
 static int irq_list[MAX_WAKEUP_REASON_IRQS];
-static int irq_count;
+static int irqcount;
 static struct kobject *wakeup_reason;
 static spinlock_t resume_reason_lock;
 
@@ -40,7 +40,7 @@ static ssize_t reason_show(struct kobject *kobj, struct kobj_attribute *attr,
 	int irq_no, buf_offset = 0;
 	struct irq_desc *desc;
 	spin_lock(&resume_reason_lock);
-	for (irq_no = 0; irq_no < irq_count; irq_no++) {
+	for (irq_no = 0; irq_no < irqcount; irq_no++) {
 		desc = irq_to_desc(irq_list[irq_no]);
 		if (desc && desc->action && desc->action->name)
 			buf_offset += sprintf(buf + buf_offset, "%d %s\n",
@@ -79,14 +79,14 @@ void log_wakeup_reason(int irq)
 		printk(KERN_INFO "Resume caused by IRQ %d\n", irq);
 
 	spin_lock(&resume_reason_lock);
-	if (irq_count == MAX_WAKEUP_REASON_IRQS) {
+	if (irqcount == MAX_WAKEUP_REASON_IRQS) {
 		spin_unlock(&resume_reason_lock);
 		printk(KERN_WARNING "Resume caused by more than %d IRQs\n",
 				MAX_WAKEUP_REASON_IRQS);
 		return;
 	}
 
-	irq_list[irq_count++] = irq;
+	irq_list[irqcount++] = irq;
 	spin_unlock(&resume_reason_lock);
 }
 
@@ -97,7 +97,7 @@ static int wakeup_reason_pm_event(struct notifier_block *notifier,
 	switch (pm_event) {
 	case PM_SUSPEND_PREPARE:
 		spin_lock(&resume_reason_lock);
-		irq_count = 0;
+		irqcount = 0;
 		spin_unlock(&resume_reason_lock);
 		break;
 	default:
-- 
1.9.1

